src/backend/routes/auth.js

```diff
@@ -37,5 +37,24 @@ module.exports = (passport) => {
     })(req, res, next);
   });
 
+  router.get('/auth/google', passport.authenticate('google', { scope: 'profile email' }));
+  router.get('/auth/google/callback', (req, res, next) => {
+    passport.authenticate('google', (err, user, info) => {
```

Comment:
The (err, user, info) function looks nearly identical to the one I remember from the auth/login. If they're close enough we should extract it out as a function "loginCallback" or something to use in both (and future) places